test/graphkb.test.js

```diff
@@ -0,0 +1,54 @@
+const { simplifyRecordsLinks } = require('../src/graphkb');
+
+describe('simplifyRecordsLinks', () => {
+    test('simplifies nested objects into their RID only', () => {
+        const listOfContentToReturnAsIs = [
+            1.0,
+            null,
+        ];
+        const listOfContentToReturnUnnested = [
+            {},
+            { a: 1, b: 2 },
+            { '@rid': 123, a: 1 },
+            {
+                a: [
+                    { '@rid': 123, aa: 1 },
+                    { ab: 2 },
+                ],
+            },
+            {
+                a: { '@rid': 123, aa: 1 },
+                b: 2,
+            },
+            {
+                a: {
+                    '@rid': 123,
+                    a: { '@rid': 123, aa: 1 },
+                },
+            },
+        ];
+        const ListOfReturnedUnnestedContent = [
+            {},
+            { a: 1, b: 2 },
+            { '@rid': 123, a: 1 },
+            {
+                a: [
+                    '123',
+                    { ab: 2 },
+                ],
+            },
+            { a: '123', b: 2 },
+            { a: '123' },
+        ];
+
+        listOfContentToReturnAsIs.forEach(content => {
```

Comment:
rather than foEach here you should use test.each (aka jest equivalent to pytest parametrize)

Reply:
see: https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Refactoring the first test with test.each has been straight forward but I struggled a bit with the second since I wanted to 'sync-compare' each input with it's own expected output. You tell me if my solution is appropriate.